src/cli.go

```diff
@@ -59,7 +60,8 @@ func newAPIClient(ua *userEndpointAuth, cache bool) *sourcegraph.Client {
 
 	var transport http.RoundTripper
 	if cache {
-		transport = http.RoundTripper(httpcache.NewTransport(diskcache.New("/tmp/srclib-cache")))
+		tempdir := path.Join(os.TempDir(), "srclib-cache")
```

Comment:
Should this use `ioutil.TempDir` instead? Wasn't sure if this needs to persist between requests or some such in which case `ioutil.TempDir`'s ability to pick a different name if the one you ask for is taken would be a bad thing.